sqlglot/dialects/duckdb.py

```diff
@@ -44,6 +44,7 @@
 from sqlglot.helper import seq_get
 from sqlglot.tokens import TokenType
 from sqlglot.parser import binary_range_parser
+from sqlglot.dialects.postgres import _regexp_replace_global_modifier
```

Comment:
Let's not `_`-prefix stuff that should be imported elsewhere. I'd move this to `dialect.py`, tbh.